scripts/bootstrap.py

```diff
@@ -0,0 +1,65 @@
+# Server Information
```

Comment:
can you think of a more descriptive name that tells me what I'm bootstrapping?

Reply:
Changed to `dbe-setup-and-run.py`, does this work for you?